src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -45,6 +47,15 @@ private static T GetService<T>(this IServiceProvider services, T defaultService)
 
         public static readonly IReadOnlyDictionary<TexlFunction, AsyncFunctionPtr> FunctionImplementations;
 
+        internal class FormattingInfo
```

Comment:
>internal class FormattingInfo [](http://example.com/codeflow?start=8&length=29)

nit - we don't need to nest this class.

Reply:
what is your suggestion here to create object for formattinginfo?